bundles/redhat8.6/bundle.sh.gotmpl

```diff
@@ -45,9 +45,19 @@ subscription::defer_unregister() {
 subscription-manager release --set=8.6
 subscription-manager refresh
 subscription::defer_unregister
-subscription-manager repos --enable rhel-8-for-x86_64-baseos-eus-rpms
-subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
-subscription-manager repos --enable rhel-8-for-x86_64-appstream-rpms
+
+ENABLED_REPOS="codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms"
```

Comment:
WDYT about putting this inside the `else`, closer to where they are enabled?

Reply:
I think its fine to leave it here, these are the default values.